shell/packages/sandstorm-db/db.js

```diff
@@ -214,6 +214,8 @@ Grains = new Mongo.Collection("grains");
 //                   time a session to this grain was opened.
 //   trashed: If present, the Date when this grain was moved to the trash bin. Thirty days after
 //            this date, the grain will be automatically deleted.
+//   ownerUnread: If true, this grain has had one or mone activity events since the owner last
```

Comment:
Also it's sorta weird that this is expressed as the inverse of the equivalent field in ApiTokens, when (from what I'm guessing) it's probably intended to serve the same purpose.  Could these be refactored into a single boolean with a matching name across the two collections?

Actually, this doesn't appear to be used anywhere else in the patch - perhaps just remove it.